### PR TITLE
BUGFIX: RAIL-1220 Pivot table is loaded forever when open to the second pivot table

### PR DESCRIPTION
**Pull request by tomasreichmann**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2083/ : SUCCESS in 33s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1541/ : FAILURE in 4m 44s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2175/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1624/ : SUCCESS in 10m 40s

File: src/components/core/base/test/VisualizationLoadingHOC.spec.tsx [hunk 1]
```diff
@@ -391,5 +391,17 @@ describe('VisualizationLoadingHOC', () => {
                 expect(onError).toHaveBeenCalledWith(new RuntimeError(ErrorStates.DATA_TOO_LARGE_TO_COMPUTE));
             });
         });
+
+        it('getPage should return promise if component has been unmounted', async () => {
```
Comment (tomassatinsky) [inline src/components/core/base/test/VisualizationLoadingHOC.spec.tsx]: If possible, tweak the text little bit. It sounds like the function does not return promise when the component has not been unmounted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/488/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1632/ : SUCCESS in 1m 45s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/879/ : SUCCESS in 19m 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/432/ : SUCCESS in 22m 20s

**Pull request merged**